webpack.config.js

```diff
@@ -174,51 +170,19 @@ supportedLocales
           : chunkGraph.getModuleChunks(module).some(c => c.name && pattern.test(c.name))
       );
 
+    // We are defining a chunk that combines the django language files with
+    // moment's locales as if you want one, you will want the other.
+    //
+    // In the application code you will still need to import via their module
+    // paths and not the chunk name
     localeChunkGroups[group] = {
-      chunks: 'initial',
+      chunks: 'async',
       name: group,
       test: groupTest,
       enforce: true,
     };
   });
 
-/**
- * Restrict translation files that are pulled in through app/translations.jsx
- * and through moment/locale/* to only those which we create bundles for via
- * locale/catalogs.json.
- */
-const localeRestrictionPlugins = [
```

Comment:
Not needed as the dynamic import will 404 and will continue in english.

Reply:
nice

Reply:
Actually had to restore this as webpack will still output these, the Ignore plugin makes sure they're not bundled together.